setup.py

```diff
@@ -40,7 +40,7 @@ def readme():
         'Topic :: Utilities',
     ],
     keywords=['doctest', 'rst', 'pytest', 'py.test'],
-    install_requires=['six', 'pytest>=3.0', 'numpy>=1.10'],
+    install_requires=['six', 'pytest>=2.8.0'],
```

Comment:
keep  the pytest requirement

Reply:
(a rebase would resolve that, I see this was coming from commit cherry-picked from the other PR)